addons/stock/models/stock_move.py

```diff
@@ -257,7 +257,10 @@ def _quantity_done_compute(self):
         """
         for move in self:
             quantity_done = 0
-            for move_line in move._get_move_lines():
+            move_lines = move._get_move_lines()
+            if self._context.get('exclude_owner'):
```

Comment:
It's a good idea but to make a more "leave stock_account shit in stock_account" patch I would override `_get_move_lines()` in stock_account and return lines without owner in case the context is filled with exclude_owner as you did

Reply:
I've updated the fix. Thanks.